simulation/simulation.py

```diff
@@ -99,12 +109,28 @@ def _update_simulation_objects(self):
             # TODO: update threshold position
             for agent in cluster.agents:
                 # TODO: move to other method -> update_agents
-                agent.body.angular_velocity = utils.get_agent_velocity(
-                    cluster.threshold.position,
-                    agent.body.position.x
-                )
+                if agent.body.position[1] < -300:
```

Comment:
Magic number. Może jakiś odpowiednik enuma żeby uniknąć nic nieznaczących liczb? (słownik? nie wiem jak to wygląda w ygląda w pythonie :/ )

Reply:
Albo jeśli to jest początek mapy to użyć zmiennej self._map_end[0]. Cokolwiek znaczącego.